scripts/watch.js

```diff
@@ -22,7 +22,7 @@ function runServer () {
     child.removeAllListeners()
   }
 
-  child = childProcess.fork(path.join(__dirname, 'start.js'), ['--devWatch'], {
+  child = childProcess.fork(path.join(__dirname, 'start.js'), [...process.argv, '--devWatch'], {
```

Comment:
Why do we need this?

Reply:
b/c we're currently not passing along the CLI args to the `start.js` script:

```
../../scripts/start.js --component-testing --project ${PWD} # gets proper argv

../../scripts/watch.js --component-testing --project ${PWD} # only gets --devWatch
```